pheonixblade9/dotnet-docs-samples
# Clean up DLP tests and add tests for List and Create jobs in new styl…

…e (#1071)

* Move DLP code samples to newer style

* Add test for ListDlpJobs and normalize line endings

* Add JobCreateTest with GCS interaction

* Add missing resource files

* Update dlp/api/DlpSample/DeIdentify.cs

Co-authored-by: Amanda Tarafa Mas <14878252+amanda-tarafa@users.noreply.github.com>

* Add PR comment fixes

* Adding namespaces back in for tests

* Removing illegal constructor

* Fix lint issues

* fixing lint issues

* Fix lint error

* fix lint

* Commenting out linter

* Add editorconfig to correctly format code in Visual Studio

* Add Lint check back in

* Revert "Add Lint check back in"

This reverts commit 28779637023f289feb941fea1b4b0276215d08c6.

Co-authored-by: Cody Herring <codedude@google.com>
Co-authored-by: Amanda Tarafa Mas <14878252+amanda-tarafa@users.noreply.github.com>
